tests/channels/conftest.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import annotations
+
+import pytest
+from redis import Redis
+from redis.asyncio import Redis as AsyncRedis
+
+from litestar.channels.memory import MemoryChannelsBackend
+from litestar.channels.redis import RedisChannelsPubSubBackend, RedisChannelsStreamBackend
+
+
+@pytest.fixture()
+def redis_client(docker_ip: str) -> AsyncRedis:
+    Redis(host=docker_ip, port=6397).flushall()
```

Comment:
For some reason pytest-asyncio goes haywire when this is an async function so we use a separate sync redis client to perform the flush

Reply:
What's the purpose for the `flushall` call here?  Without storage allocated to the container, shouldn't the Redis database be empty already?

Reply:
The redis service is session scoped. This fixture is function scoped. By calling `flushall` it’s ensured that every test invocation gets an empty database.